src/config/config_generator.cc

```diff
@@ -374,42 +359,42 @@ void ConfigGenerator::generateOnlineContent()
+    setValue(oggmp3, ATTR_TRANSCODING_PROFILES_PROFLE_MIMETYPE, "audio/mpeg");
+    setValue(oggmp3, ATTR_TRANSCODING_PROFILES_PROFLE_ACCURL, NO);
+    setValue(oggmp3, ATTR_TRANSCODING_PROFILES_PROFLE_FIRST, YES);
+    setValue(oggmp3, ATTR_TRANSCODING_PROFILES_PROFLE_ACCOGG, NO);
 
-    auto agent = setValue("/transcoding/profiles/profile/agent/", "", true);
+    auto agent = setValue(std::string(ConfigManager::mapConfigOption(ATTR_TRANSCODING_PROFILES_PROFLE)) + "/agent/", "", true);
     setValue(agent, ATTR_TRANSCODING_PROFILES_PROFLE_AGENT_COMMAND, "ffmpeg");
     setValue(agent, ATTR_TRANSCODING_PROFILES_PROFLE_AGENT_ARGS, "-y -i %in -f mp3 %out");
 
-    auto buffer = setValue("/transcoding/profiles/profile/buffer/", "", true);
+    auto buffer = setValue(std::string(ConfigManager::mapConfigOption(ATTR_TRANSCODING_PROFILES_PROFLE)) + "/buffer/", "", true);
     setValue(buffer, ATTR_TRANSCODING_PROFILES_PROFLE_BUFFER_SIZE, std::to_string(DEFAULT_AUDIO_BUFFER_SIZE));
     setValue(buffer, ATTR_TRANSCODING_PROFILES_PROFLE_BUFFER_CHUNK, std::to_string(DEFAULT_AUDIO_CHUNK_SIZE));
     setValue(buffer, ATTR_TRANSCODING_PROFILES_PROFLE_BUFFER_FILL, std::to_string(DEFAULT_AUDIO_FILL_SIZE));
 
-    auto vlcmpeg = setValue("/transcoding/profiles/profile/", "", true);
+    auto vlcmpeg = setValue(std::string(ConfigManager::mapConfigOption(ATTR_TRANSCODING_PROFILES_PROFLE)) + "/", "", true);
     setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_NAME, "vlcmpeg");
     setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_ENABLED, NO);
     setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_TYPE, "external");
-    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_MIMETYPE, "video/mpeg", true);
-    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_ACCURL, YES, true);
-    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_FIRST, YES, true);
-    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_ACCOGG, YES, true);
+    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_MIMETYPE, "video/mpeg");
+    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_ACCURL, YES);
+    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_FIRST, YES);
+    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_ACCOGG, YES);
 
-    agent = setValue("/transcoding/profiles/profile/agent/", "", true);
+    agent = setValue(std::string(ConfigManager::mapConfigOption(ATTR_TRANSCODING_PROFILES_PROFLE)) + "/agent/", "", true);
     setValue(agent, ATTR_TRANSCODING_PROFILES_PROFLE_AGENT_COMMAND, "vlc");
     setValue(agent, ATTR_TRANSCODING_PROFILES_PROFLE_AGENT_ARGS, "-I dummy %in --sout #transcode{venc=ffmpeg,vcodec=mp2v,vb=4096,fps=25,aenc=ffmpeg,acodec=mpga,ab=192,samplerate=44100,channels=2}:standard{access=file,mux=ps,dst=%out} vlc:quit");
 
-    buffer = setValue("/transcoding/profiles/profile/buffer/", "", true);
+    buffer = setValue(std::string(ConfigManager::mapConfigOption(ATTR_TRANSCODING_PROFILES_PROFLE)) + "/buffer/", "", true);
```

Comment:
maybe a fmt::format would be nicer here?

Reply:
Leftovers... Cleaned them out.